pkg/ipc/ipc.go

```diff
@@ -95,8 +95,9 @@ type CallInfo struct {
 }
 
 type ProgInfo struct {
-	Calls []CallInfo
-	Extra CallInfo // stores Signal and Cover collected from background threads
+	Calls      []CallInfo
+	Extra      CallInfo // stores Signal and Cover collected from background threads
+	ElapsedSec float64  // total execution time in seconds
```

Comment:
I would store this as time.Duration. Floats are pretty specific thing, if something wants floats (learning), it should convert the standard time to floats.
Also if we add per-syscall execution time, storing it as seconds will be somewhat strange since most syscalls execute within microseconds.